marquez_client/utils.py

```diff
@@ -99,3 +99,9 @@ def utc_now():
     def get_json(file):
         with open(file) as json_file:
             return json.load(json_file)
+
+    @staticmethod
+    def handle_fail(cause):
+        log.debug(f"cause: {cause}")
```

Comment:
This feels more like a message we'd want to log as an error. We can also pass [`exc_info=cause`](https://docs.python.org/2/library/logging.html#logrecord-objects) to `log.error()`, which handles formatting the exception info.

Reply:
yeah, and also I feel that it would be better to pass a lambda expression where you use this. Utils.handle_fails is not a clear name of what this is meant for.,

Reply:
this method is passed to jsonpickle.encode(value=object, fail_safe=handle_fail)
whenever a failure occurs when serializing the object, the message is captured and added as a value for that key.
eg. in the description, above.